types/aws-lambda/test/api-gateway-tests.ts

```diff
@@ -194,6 +227,24 @@ function createProxyResult(): APIGatewayProxyResult {
     return result;
 }
 
+function createProxyResultV2(): APIGatewayProxyResult {
```

Comment:
Function for v1 was mostly because it was re-used, so you could inline this... obv. not a big deal.